test/jdk/java/nio/channels/Channels/TransferTo.java

```diff
@@ -154,14 +156,29 @@ public void testStreamContents(InputStreamProvider inputStreamProvider,
      */
     @Test
     public void testMoreThanTwoGB() throws IOException {
-        Path sourceFile = Files.createTempFile(CWD, "test2GBSource", null);
+        Path sourceFile = CWD.resolve(Path.of(String.format("test3GBSource%s.tmp",
+            String.valueOf(RND.nextInt(Integer.MAX_VALUE)))));
```

Comment:
The Path.of isn't needed here as there is an overload of resolve that takes a String. For readability then it might be simpler to create the file name (as a String) on its own line.